modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -548,6 +664,100 @@ private CompletableFuture<Replica> startReplicaInternal(
                 .thenCompose(v -> replicaFuture);
     }
 
+    /**
+     * TODO: will be private after https://issues.apache.org/jira/browse/IGNITE-22315
+     * Temporary public method for RAFT-client starting.
+     *
+     * @param replicaGrpId Replication Group ID.
+     * @param newConfiguration Peers and learners nodes for a raft group.
+     * @param raftClientCache Temporal supplier that returns RAFT-client from TableRaftService if it's already exists and was put into the
```

Comment:
Pls, add TODO with ticket for removal

Reply:
But there is above: 
>TODO: will be private after https://issues.apache.org/jira/browse/IGNITE-22315

Maybe better to change `private` to `removed`?

Reply:
Changed to removed